activation/handler_v2.go

```diff
@@ -713,9 +712,34 @@ func (h *HandlerV2) checkDoubleMarry(
 			return false, fmt.Errorf("checking if ID is married: %w", err)
 		}
 		if mATX != atxID {
-			// TODO(mafa): finish proof
-			proof := &wire.ATXProof{
-				ProofType: wire.DoubleMarry,
+			var blob sql.Blob
+			v, err := atxs.LoadBlob(ctx, tx, atxID.Bytes(), &blob)
```

Comment:
We have this ATX in wire form - it's the one being processed. It would be simpler just to pass it in.

Reply:
Updated